testing/src/cypress/e2e/02-institution/02-dashboard-institution-page.cy.ts

```diff
@@ -6,7 +6,18 @@ const dashboardObject = new DashboardInstitutionObject();
 const manageInstitutionObject = new ManageInstitutionObject();
 const institutionHelperActions = new InstitutionHelperActions();
 
-describe("[Institution Dashboard] - fields and titles ", () => {
+enum Locations {
+  Vancouver = "Vancouver",
```

Comment:
mm..why is this enum for

Reply:
Since we control the data seeding in, we can use these enums to get to any of the locations desired. Like viewing/editing those specific locations etc

Reply:
but, i didn't find it being used

Reply:
Right, removed it.